Ginger/GingerCoreNET/BPMN/ActivitiesGroupToBPMNConverter.cs

```diff
@@ -132,23 +178,30 @@ private IEnumerable<Activity> GetActivitiesFromActivityGroup(ActivitiesGroup act
                     .Select(identifier => identifier.IdentifiedActivity);
         }
 
-        private string GetTargetAppNameFromConsumerId(Consumer consumer)
+        private TargetBase GetTargetAppFromConsumer(Consumer consumer)
         {
             IEnumerable<TargetBase> targetApps = WorkSpace.Instance.Solution.GetSolutionTargetApplications();
             TargetBase? consumerTargetApp = targetApps.FirstOrDefault(targetApp => string.Equals(targetApp.Guid, consumer.ConsumerGuid));
             if (consumerTargetApp == null)
             {
-                throw new InvalidOperationException($"No Target Application found for Consumer with Guid '{consumer.ConsumerGuid}'.");
+                throw new BPMNExportException($"No Target Application found for Consumer with Guid '{consumer.ConsumerGuid}'.");
             }
-            return consumerTargetApp.Name;
+            return consumerTargetApp;
+        }
+
+        private TargetBase? GetTargetAppFromTargetAppName(string targetAppName)
+        {
+            return WorkSpace.Instance.Solution
+                .GetSolutionTargetApplications()
+                .FirstOrDefault(targetApp => string.Equals(targetApp.Name, targetAppName));
         }
 
         private Participant GetParticipantForTargetAppName(Collaboration collaboration, string targetAppName)
         {
             Participant? participant = collaboration.Participants.FirstOrDefault(participant => string.Equals(participant.Name, targetAppName));
             if(participant == null)
             {
-                throw new InvalidOperationException($"No Participant found for Target Application name '{targetAppName}'.");
+                throw new BPMNExportException($"No Participant found for Target Application name '{targetAppName}'.");
```

Comment:
Use Ginger Resource dictionary to show entity names according to the selected terminology,
Also add corresponding Ginger entity for participant